house/bluetooth.go

```diff
@@ -0,0 +1,75 @@
+package house
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+type bleDevice struct {
+	Id       string    `json:"id",yaml:"id"`
+	LastSeen int64     `json:"lastSeen",yaml:"lastSeen"`
+	Commands []command `json:"commands",yaml:"commands"`
+	Name     string    `json:"name",yaml:"name"`
+}
+
+type command struct {
+	Timeout        int64  `json:"timeout",yaml:"timeout"`
+	Command        string `json:"command",yaml:"command"`
+	TimeoutCommand string `json:"timeoutcommand",yaml:"timeoutcommand"`
+}
+
+type TimedCommand struct {
```

Comment:
warning: exported type TimedCommand should have comment or be unexported (golint)